terraform/scheduler.tf

```diff
@@ -42,4 +42,12 @@ resource "google_cloud_scheduler_job" "orama_deploy" {
       "Authorization" = "Bearer ${var.orama_package_private_api_key}"
     }
   }
+
+  http_target {
```

Comment:
I don't think you can have two targets. I think you need to create a second `google_cloud_scheduler_job`

Reply:
ah damn, yea wasnt sure, was hoping it would work